docs/PacketFence_JSON-RPC_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,50 @@
+Generic firewall (JSON-RPC) Quick Integration Guide
+===================================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+:toc: left
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using a firewall with a *JSON-RPC* interface.
+
+Assumptions
+-----------
+* You have a configured PacketFence environment with working test equipment;
+* You have a firewall with a compatible JSON-RPC interface (see below).
+
+JSON-RPC interface
+------------------
+The JSONRPC module shipped with PacketFence is meant as a generic firewall SSO module to be used with Linux or BSD firewalls that do not by default ship with a vendor-specific interface to do SSO with.
+
+A compatible server must implement the methods `Start` and `Stop`, both with the identical set of parameters provided below.
+
+[options="compact"]
+* *Protocol*: JSON-RPC 2.0 over HTTPS
+* *Authentification*: HTTP Basic authentication
+* *Methods*: `Start` and `Stop`
+* *Parameters*:
+  - `user`    _(string)_: Username that registered the device
+  - `mac`     _(string)_: MAC address of the device
+  - `ip`      _(string)_: IP address of the device
+  - `role`    _(string)_: PacketFence role assigned to the device
+  - `timeout` _(int)_:    Duration until the registration expires in seconds
+* *Response*: Success must be indicated by `"result": ["OK"]`. Every string other than `OK` is taken as an error message.
+
+A simple JSON-RPC server written in Python that is compatible with this specification and creates ipsets based on the SSO information provided by PacketFence can be found at https://github.com/tribut/ipset-rpcd.
+
+SSO Configuration in PacketFence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Go to *'Configuration' -> 'Firewall SSO' -> 'Add Firewall' -> 'JSONRPC'*.
```

Comment:
The path should be "Configuration -> Integration -> Firewall SSO -> ..." (from 7.0 and onward)